docs/HowTo.txt

```diff
@@ -450,3 +450,26 @@ e.g.
     # Select an item in the popup menu
     outlook.PopupMenu.Menu().get_menu_path("Cancel Server Request")[0].click()
 
+
+
+COM Model
+------------------------------------------------------------------------------------
+By default, pywinauto sets up the client Multithreading COM model (MTA) on init if
+no other model was defined prior to import of pywinauto. The model can be set up by
+another imported module implicitly or specified explicitly through ```sys.coinit_flags```
```

Comment:
As I remember double "code" quotes is enough for inline code in ReSTructured text.